app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -285,7 +285,7 @@ def assister_index
   end
 
   def find_email(agent, role)
-    if role == 'Broker'
+    if role == l10n("broker")
```

Comment:
No! If we have changed what the forms submit we will break a lot of stuff, these are still Brokers in the DB, models and classes.